shell_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"crypto/md5"
 	"fmt"
+	files "github.com/ipfs/go-ipfs-files"
```

Comment:
why not "go-btfs-files"

Reply:
i tried but it reports errors like no path.. it may be related to go-btfs-api's lower go-ipfs-files v0.0.1.

Reply:
Yeah, it should not have any lower go-ipfs-files dependencies.. otherwise we have not truly fixed the folder issue.

Reply:
Updated the imports.